cypress/e2e/hw18.cy.js

```diff
@@ -0,0 +1,68 @@
+import MainPageSteps from '../support/steps/MainPageSteps';
+
+describe(`HW 18.1. Cypress inquiries`, () => {
+  beforeEach(() => {
+    cy.fixture('user').as('userData');
+    cy.visit("/");
+    cy.get(".hero-descriptor_btn").click();
+  });
+
+  it(`Checking Name`, () => {
+    MainPageSteps.RegistrationFormNameEmpty(),
```

Comment:
You have actions and assertions in the same method

It is not really good practice